[DOCS] Kibana 9.1.0 release notes (#227077)
elastic/kibana: docs/release-notes/index.md

* Updates time-based charts to use the multi-layer time axis by default, providing a better time window context and improved label positioning [#210579]({{kib-pull}}210579).
* Enables read-only editor mode to inspect a panel's configuration in Lens [#208554]({{kib-pull}}208554).
* Adds a **Point visibility** option to Area and Line charts in Lens [#222187]({{kib-pull}}222187).
* Adds a setting to control the data table density in Lens [#220252]({{kib-pull}}220252).
* Opens Lens in the same tab when you select **Open in Lens** on a visualization [#217528]({{kib-pull}}217528).
* Adds the ability to open links from Vega visualizations in a new tab by specifying the `"target": "_blank"` option for the `usermeta.embedOptions.loader` property of the Vega chart configuration [#216200]({{kib-pull}}216200).
* Adds globe projection improvements to Maps [#212437]({{kib-pull}}212437).
* When possible, keeps the chart configuration after editing an {{esql}} visualization's query [#210780]({{kib-pull}}210780).

**Data ingestion and Fleet**:
* Adds support for bulk agent migration using the **Bulk actions** menu in the agent list table [#224334]({{kib-pull}}224334).

> * Enables the `Tabular integrations` UI feature flag [#222842]({{kib-pull}}222842).

Reviewer: ```suggestion
* Enables the tabular integrations feature in Fleet [#222842]({{kib-pull}}222842).
```

* Adds support for single agent migration using the **Actions** menu in {{fleet}}. Users can provide a remote cluster URL and enrollment token, as well as customize parameters for the migration [#222111]({{kib-pull}}222111).
* Adds an API endpoint to migrate a single agent to another cluster using a URL and enrollment token. Tamper-protected and {{fleet}}-managed agents are not supported and return a `403` response if attempted [#220601]({{kib-pull}}220601).
* Adds a new integration flyout component [#220229]({{kib-pull}}220229).

> * Enables the `enableSyncIntegrationsOnRemote` feature flag [#220215]({{kib-pull}}220215).

Reviewer 2: ```suggestion
* Enables the sync integrations on remote clusters feature in Fleet [#220215]({{kib-pull}}220215).
```

* Adds SSL options to {{fleet}} Server host settings [#208091]({{kib-pull}}208091).
* Adds a new action to add tags from the **Agent details** page [#225433]({{kib-pull}}225433).
* Adds a tooltip to the **Last activity** column in the agent list UI [#224850]({{kib-pull}}224850).
* Adds support for agentless traffic filters [#222082]({{kib-pull}}222082).
* Makes the tag filter in {{fleet}} agents searchable and sorted [#219639]({{kib-pull}}219639).
* Adds a callout to highlight breaking changes during integration upgrades [#217257]({{kib-pull}}217257).
* Adds MSI installer command support for {{fleet}} Server and agents [#217217]({{kib-pull}}217217).
* Makes SSL optional for {{fleet}} Logstash outputs [#216216]({{kib-pull}}216216).
* Formats the **Last activity** value in the {{fleet}} agent details view as a datetime [#215531]({{kib-pull}}215531).
* Adds support for `searchAfter` and point-in-time (`pit`) parameters in the get agents list API [#213486]({{kib-pull}}213486).
* Registers a custom integrations search provider [#213013]({{kib-pull}}213013).
* Adds support for collapsible sections in integration overview pages [#223916]({{kib-pull}}223916).

**Discover**:

> * Adds an **Attributes** tab in **Discover** when exploring OTel documents [#222391]({{kib-pull}}222391).

Reviewer 3: ```suggestion
* Adds an **Attributes** tab to the document viewer when exploring OTel documents in the Observability solution view [#222391]({{kib-pull}}222391).
```

* When an ES|QL query times out (as a result of the `search:timeout` advanced setting), partial results that are available are now shown [#219027]({{kib-pull}}219027).

> * Adds click actions for Stacktrace and Degraded Fields in Discover [#214413]({{kib-pull}}214413).

Reviewer 3: ```suggestion
* Adds click actions for Stacktrace and Degraded Fields when exploring logs in Discover [#214413]({{kib-pull}}214413).
```

* Shows a **Load more** option instead of pagination when exploring Logs in **Discover** [#211176]({{kib-pull}}211176).
* Expands the {{esql}} editor to fit the query size automatically when loading **Discover** [#225509]({{kib-pull}}225509).
* Hides the "Selected only" toggle in **Discover**'s pages that don't support filtering by value [#220624]({{kib-pull}}220624).
* Adds a **Copy value** button to field value cells in the Document viewer [#218817]({{kib-pull}}218817).
* Adds a warning and a tooltip for explaining the `_score` column in **Discover** [#211013]({{kib-pull}}211013).

> * Adds support for `command`/`ctrl` + click to open links in a new tab, for example allowing to conduct simultaneous searches in parallel more efficiently [#210982]({{kib-pull}}210982).

Reviewer 3: ```suggestion
* Adds support for `command`/`ctrl` + click to open new Discover sessions in a separate tab, for example allowing to conduct simultaneous searches in parallel more efficiently [#210982]({{kib-pull}}210982).
```

* Replaces registry call when deleting {{kib}} assets for custom packages [#224886]({{kib-pull}}224886).
* Ensures package policy names are unique when moving across spaces [#224804]({{kib-pull}}224804).
* Fixes bulk actions incorrectly selecting agents when a namespace filter is applied [#224036]({{kib-pull}}224036).
* Reverts "Added instructions for installing {{agent}} complete." [#223520]({{kib-pull}}223520).
* Fixes styled component theme lookup issue [#221979]({{kib-pull}}221979).
* Fixes SSL config being overridden by advanced YAML in full agent policy [#219902]({{kib-pull}}219902).
* Fixes required capability for the SIEM migrations topic [#219427]({{kib-pull}}219427).
* Makes output and {{fleet}} Server settings non-editable for agentless policies [#218905]({{kib-pull}}218905).
* Supports integrations with secrets that contain multiple values [#216918]({{kib-pull}}216918).
* Adds remote cluster instructions for syncing integrations [#211997]({{kib-pull}}211997).
* Updates install snippets to include all platforms [#210249]({{kib-pull}}210249).

**Discover**:
* Fixes invalid input highlight in **Data View** flyout [#226822]({{kib-pull}}226822).

> * Fixes an issue causing an internal server error when renaming a search session [#226757]({{kib-pull}}226757).

Reviewer 3: This one was a dev mode only bug, not sure if we usually include those kinds of things or not, so just mentioning.

Reviewer 4: ```suggestion
```

* Fixes an issue causing **Discover** to freeze when dragging & dropping columns with animations disabled [#226592]({{kib-pull}}226592).
* Fixes row highlighting when reordering columns [#226584]({{kib-pull}}226584).
* Fixes an issue causing an error when updating then deleting a saved query [#226569]({{kib-pull}}226569).

> * Fixes a cell value alignment issue [#226562]({{kib-pull}}226562).

Reviewer 3: ```suggestion
* Fixes a cell value alignment issue for aggregate metric fields [#226562]({{kib-pull}}226562).
```

* Adds missing information icon to the document viewer table [#222299]({{kib-pull}}222299).
* Fixes an issue incorrectly showing the *unmapped* icon when a field changed from unmapped to mapped [#221308]({{kib-pull}}221308).
* Fixes the parsing of index patterns in the **Inspect** feature of Kibana. Previously, certain index pattern strings were not being parsed and displayed correctly in the **Inspect** feature [#221084]({{kib-pull}}221084).

> * Fixes an issue that causes transitions from **Logs Stream** and **Logs Explorer** to **Discover** to lose some context such as the selected time range or KQL query [#215867]({{kib-pull}}215867).

Reviewer 3: ```suggestion
* Fixes an issue that causes redirects from the deprecated **Logs Stream** and **Logs Explorer** routes to **Discover** to lose some context such as the selected time range or KQL query [#215867]({{kib-pull}}215867).
```

* Excludes only {{es}} metadata fields from the Summary column instead of all fields starting with `_` [#213255]({{kib-pull}}213255).
* Fixes inability to clear Document ID in data view field editor preview [#220891]({{kib-pull}}220891).

> * Fixed multiple accessibility issues, including adding missing aria labels and column headers, improving keyboard navigation and interactions, and improving focus changes when interacting with **Discover** features [View list of fixes](https://github.com/elastic/kibana/issues?q=state:closed%20label:Project:Accessibility%20label:v9.1.0%20label:Team:DataDiscovery).

Reviewer 4: @kertal @davismcphee this is the a11y fixes summary we had talked about over Slack, lmk how that sounds to you. I added the `Project:Accessibility` label to a few PRs that were missing it so I could link to the list of a11y fixes with a single link.

Reviewer 3: Looks great! Thanks @florent-leborgne 🙏
